decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative('./nameable')
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
```

Comment:
- [ ] [REQUIRED] Please add `attr_accessor :nameable` after line `3`, also it would be nice remove `super()` 🙏🏼

Reply:
It gives me an error if I don't instantiate the parent class.